tests/php/Security/MemberTest.php

```diff
@@ -1198,18 +1199,21 @@ public function testRememberMeMultipleDevices()
             ]
         );
         $this->assertEquals(
+            1,
             RememberLoginHash::get()->filter(['MemberID'=>$m1->ID, 'DeviceID'=>$firstHash->DeviceID])->count(),
-            1
         );
 
-        // Logging out from any device when all login hashes should be removed
-        RememberLoginHash::config()->update('logout_across_devices', true);
-        Injector::inst()->get(IdentityStore::class)->logIn($m1, true);
-        $this->get('Security/logout', $this->session());
-        $this->assertEquals(
-            RememberLoginHash::get()->filter('MemberID', $m1->ID)->count(),
-            0
-        );
+        // If session-manager module is installed then logout_across_devices is modified so skip
+        if (!class_exists(LoginSession::class)) {
```

Comment:
Is the behaviour when session manager installed predictable? Maybe we want to add an opposing test here or in session manager.